index.html

```diff
@@ -1,3 +1,114 @@
+<!-- INSERTED BY PRABESH -->
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./styles/style.css">
+    <title>Bikes For Refugees - CYF</title>
+</head>
+<body>
+   <header>
+       <div class="logo-container flex-item">
+           <img src="./images/logo.png" class="logo-img" alt="logo-image">
+       </div>
+       <ul class="navigation-ul flex-item">
+           <li><a href="">HOME</a></li>
+           <li><a href="">ABOUT US</a></li>
+           <li><a href="">NEWS & EVENTS</a></li>
+           <li><a href="">CONTACT US</a></li>
+       </ul>
+       <button class="donate-now flex-item">Donate Now</button>
+   </header>
+   <div class="thankyou-message-display-box">
+       <p>You've helped us give 72 bikes to refugees and asylum seekers. <strong>Thank you!</strong></p>
+   </div>
+   <section class="main-body-content">
+       <div class="left-section">
```

Comment:
How would you make this html more semantic?  A number of places where this could be improved...